src/LDAP/Schema/Util.hs

```diff
@@ -78,13 +78,13 @@ makeAttributeType name str =
         , ClassD
             []
             ("MustHave" <> name)
-            [PlainTV a ()]
```

Comment:
This broke compilation for me. `PlainTV` takes two parameters: https://hackage.haskell.org/package/template-haskell-2.19.0.0/docs/Language-Haskell-TH.html#v:PlainTV

Reply:
Oh shoot, thanks for checking that. I was using ghc 8.10.7 with [template-haskell 2.16.0.0](https://hackage.haskell.org/package/template-haskell-2.16.0.0/docs/Language-Haskell-TH.html#v:PlainTV).

Will using the existing nix config get me the right pinned versions? I can switch over to a more recent ghc and lib regardless.

Reply:
That is indeed what the Nix config is for. :slightly_smiling_face:

You can either enter a shell with `nix develop` in which you can run `cabal` directly; or you can try `nix build -L`/`nix run -L` to build/run the package in the Nix sandbox.